### PR TITLE
WB-1456 fix individual page to display deprecated incremented project…

### PR DESCRIPTION
**Pull request by Atticus29**
… IDs as merged and prevent them from displaying

### CODE REVIEW

File: src/main/java/org/ecocean/servlet/MergeIndividual.java [hunk 1]
```diff
@@ -145,14 +150,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             for (int i=0; i<desiredIncrementalIdArr.size(); i++){
               if(!deprecatedIncrementIdsArr.get(i).equals("_")){
                 //there is a deprecated incremental ID that we need to rename and add to both individuals
-                mark1.addName("Merged " + projIdsArr.get(i),deprecatedIncrementIdsArr.get(i));
+                mark1.addName("Merged " + projNamesArr.get(i),deprecatedIncrementIdsArr.get(i));
```
Comment (naknomum) [inline src/main/java/org/ecocean/servlet/MergeIndividual.java:153]: is there a check in here to make sure that `projNamesArr.size()` is **>=** `desiredIncrementalIdArr.size()` ?  seems like this `get(i)` will run risk of `i` being out of range if not.  i guess same goes for `deprecatedIncrementalIdsArr.get(i)` as well.  might be good to sanity-check that when the parameters are read in.

**Comment by Atticus29** [comment]
@naknomum what do you think now?

**Review by naknomum — APPROVED** [review]
looks good

**Review by naknomum — APPROVED** [review]
lgtm

**Pull request merged**